docs/mkdocs.yml

```diff
@@ -26,6 +26,13 @@ extra_css:
 #       redirect_maps:
 #         tutorials/faq: config/troubleshooting/faq.md
 
+# We do not use `mike`, but enabling this will enable the version selector UI.
+# It references `versions.json` on `gh-pages` branch,
+# however we have a basic setup that only matches `version` to a subdirectory.
```

Comment:
yaml does not render code either or am i wrong?

Reply:
? It's a comment, so no it won't render differently unless you've got software to do so. It is just habit for me when referring to something that I usually would with markdown.